packages/components/src/Loader/Loader.scss

```diff
@@ -1,3 +1,28 @@
 .tc-loader {
 	display: flex;
 }
+
+.tc-app-loader-icon {
+	position: absolute;
+	width: 5em;
+	height: 7em;
+	margin-left: 14px;
+	margin-right: 10px;
+
+	animation: fadeIn 2s linear infinite;
```

Comment:
could you reuse these ones or add it [here](https://github.com/Talend/ui/blob/master/packages/theme/src/theme/_animation.scss)

Reply:
fixed